mautic/mautic
# Configurable assets stream extensions

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | https://github.com/mautic/user-documentation/pull/192
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
Follow up for https://github.com/mautic/mautic/pull/11043
Some extensions have to be download everytime, for example ICS. 
Then we've added list of extensions to display in browser by default. Others are download as before



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create PDF asset
3. Check If download link open the PDF
4. Create CSV asset
5. Check If download link download the CSV
